caffe2/CMakeLists.txt

```
@@ -299,7 +299,12 @@ if(NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
     # easier to other targets that depend on this. However, in that case, by not
     # setting the USE_TENSORPIPE compile definition, this target will just end
     # up being empty. Downstream targets should also add a #ifdef guard.
-    add_library(tensorpipe_agent "${TORCH_SRC_DIR}/csrc/distributed/rpc/tensorpipe_agent.cpp" "${TORCH_SRC_DIR}/csrc/distributed/rpc/tensorpipe_agent.h")
+    add_library(tensorpipe_agent
+      "${TORCH_SRC_DIR}/csrc/distributed/rpc/tensorpipe_agent.cpp"
+      "${TORCH_SRC_DIR}/csrc/distributed/rpc/tensorpipe_agent.h"
+      "${TORCH_SRC_DIR}/csrc/distributed/rpc/tensorpipe_utils.cpp"
+      "${TORCH_SRC_DIR}/csrc/distributed/rpc/tensorpipe_utils.h"
```

There is no need to add header files to the library, is there?
Also, if you expect them to grow, can they be added as list to variables.bzl and then read using append_to_filelist macro?
```suggestion
      "${TORCH_SRC_DIR}/csrc/distributed/rpc/tensorpipe_agent.cpp"
      "${TORCH_SRC_DIR}/csrc/distributed/rpc/tensorpipe_utils.cpp"
```

---

As `"${TORCH_SRC_DIR}/csrc/distributed/rpc/tensorpipe_agent.h"` was already there I just followed the usage, but I can remove both headers. We don't expect that list to grow though.